<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColorMapEditor</class>
 <widget class="QDialog" name="DialogColorMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>106</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>106</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>106</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ColorMapping Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Mapping</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>22</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>22</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>22</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>37</y>
     <width>359</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,1,1,1,1,1">
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>144</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	border: 1px solid #bebebe;
	padding: 1px 18px 1px 3px;
	font: 75 10pt;
	
}</string>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>custom</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/custom.png</normaloff>:/mainWindow/images/custom.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>accents</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/accents.png</normaloff>:/mainWindow/images/accents.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>color1</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/color1.png</normaloff>:/mainWindow/images/color1.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>color2</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/color2.png</normaloff>:/mainWindow/images/color2.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>color3</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/color3.png</normaloff>:/mainWindow/images/color3.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>color4</string>
       </property>
       <property name="icon">
        <iconset resource="mainWindow.qrc">
         <normaloff>:/mainWindow/images/color4.png</normaloff>:/mainWindow/images/color4.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>36</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background: rgba(229, 229, 229, 0)}</string>
      </property>
      <property name="text">
       <string>RGBA:</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_R">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_G">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_B">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_A">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>255</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="myColorButtonV" name="colorButtonV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>22</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myColorButtonV</class>
   <extends>QPushButton</extends>
   <header location="global">myColorButtonV.h</header>
   <slots>
    <signal>setColorValue(int,int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>DialogColorMapEditor</receiver>
   <slot>getChangedText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorButtonV</sender>
   <signal>setColorValue(int,int,int)</signal>
   <receiver>DialogColorMapEditor</receiver>
   <slot>setColorValue(int,int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_A</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogColorMapEditor</receiver>
   <slot>changeTransparency(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getChangedText(QString)</slot>
  <slot>setColorValue(int,int,int)</slot>
  <slot>changeTransparency(QString)</slot>
 </slots>
</ui>
